PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Pull request title
+------------------
+
+The format of the Pull request title must be:
+
+    <component>: <short_change_message> #<issue number>
```

Comment:
Now having used this since a few days, I wonder if
`<component> #<issue number>: <short message>`
is not a better format, because it looks weird in the pull request as we have two hashtag numbers after the other. (Issue and pull request number)

Reply:
no opinion, as long as all information is there. I've seen few projects with issue at the end but The question is where to put the keyword, e.g,.  <component> Fix #<issue number>: <short message>

Reply:
True, keyword only works on the end then. So lets keep it like that.